ksql-cli/src/test/java/io/confluent/ksql/util/TabularRowTest.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.util;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import org.junit.Test;
+
+public class TabularRowTest {
+
+  @Test
+  public void shouldFormatHeader() {
+    // Given:
+    final List<String> header = ImmutableList.of("foo", "bar");
+
+    // When:
+    final String formatted = new TabularRow(20, header, null).toString();
+
+    // Then:
+    assertThat(formatted, is(""
+        + "|foo     |bar     |\n"
+        + "+════════+════════+"));
+  }
+
+  @Test
+  public void shouldMultilineFormatHeader() {
+    // Given:
+    final List<String> header = ImmutableList.of("foo", "bar is a long string");
+
+    // When:
+    final String formatted = new TabularRow(20, header, null).toString();
+
+    // Then:
+    assertThat(formatted, is(""
+        + "|foo     |bar is a|\n"
+        + "|        | long st|\n"
+        + "|        |ring    |\n"
+        + "+════════+════════+"));
+  }
+
+  @Test
+  public void shouldFormatRow() {
+    // Given:
+    final List<String> header = ImmutableList.of("foo", "bar");
+
+    // When:
+    final String formatted = new TabularRow(20, header, header).toString();
+
+    // Then:
+    assertThat(formatted, is(""
+        + "|foo     |bar     |\n"
+        + "+--------+--------+"));
+  }
+
+  @Test
+  public void shouldMultilineFormatRow() {
```

Comment:
Add test for what happens if the number of columns is higher than the width....

Interesting edge case!

Reply:
😨 good catch! I think if this is the case I'll default back to the old behavior

Reply:
I've decided to set a minimum cell width of 5 - it'll look all weird if the terminal is super small, but that's going to be the case anyway